src/components/media-player/dialog-media-player-group.ts

```diff
@@ -0,0 +1,155 @@
+import type { CSSResultGroup } from "lit";
+import { mdiClose } from "@mdi/js";
+import { LitElement, css, html, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import type { HassEntity } from "home-assistant-js-websocket";
+import { fireEvent } from "../../common/dom/fire_event";
+import { haStyleDialog } from "../../resources/styles";
+import type { HomeAssistant, ValueChangedEvent } from "../../types";
+import "../ha-dialog";
+import "../ha-dialog-header";
+import "../entity/ha-entities-picker";
+import type { MediaPlayerGroupDialogParams } from "./show-media-player-group-dialog";
+import { computeStateName } from "../../common/entity/compute_state_name";
+import { supportsFeature } from "../../common/entity/supports-feature";
+import {
+  MediaPlayerEntityFeature,
+  mediaPlayerJoin,
+} from "../../data/media-player";
+
+const includeDomains = ["media_player"];
+
+@customElement("dialog-media-player-group")
+class DialogMediaPlayerGroup extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _entityId?: string;
+
+  @state() private _groupMembers!: string[];
+
+  @state() private _submitting?: boolean;
+
+  public showDialog(params: MediaPlayerGroupDialogParams): void {
+    this._entityId = params.entityId;
+
+    const stateObj = this.hass.states[params.entityId] as
+      | (HassEntity & { attributes: { group_members?: string[] } })
+      | undefined;
+    this._groupMembers =
+      stateObj?.attributes.group_members?.filter(
+        (entityId) => entityId !== params.entityId
+      ) || [];
+  }
+
+  public closeDialog() {
+    this._entityId = undefined;
+    this._groupMembers = [];
+    this._submitting = false;
+    this.classList.remove("opened");
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  protected render() {
+    if (!this._entityId) {
+      return nothing;
+    }
+
+    const entityId = this._entityId;
+    const stateObj = this.hass.states[entityId] as HassEntity | undefined;
+    const name = (stateObj && computeStateName(stateObj)) || entityId;
+
+    return html`
+      <ha-dialog
+        open
+        scrimClickAction
+        escapeKeyAction
+        flexContent
+        .heading=${name}
+        @closed=${this.closeDialog}
+        @opened=${this._dialogOpened}
+      >
+        <ha-dialog-header show-border slot="heading">
+          <span slot="title"> ${name} </span>
```

Comment:
We should give it another name than the name of the media player. 
The title should be "Group" I suppose or "Group members" although I'm doubt about it.